src/Consumer/AbstractConsumer.php

```diff
@@ -76,10 +76,15 @@ protected function getManager(): EntityManagerInterface
         return $this->manager;
     }
 
+    protected function allowExtraFields(): bool
```

Comment:
`isExtraFieldsAllowed` would be a better name IMO.
This one looks an "action" and/or "behavior" if you know what I mean.

Reply:
i know what you mean.

Reply:
agree ;)